### PR TITLE
Adds Quote Service to Demo Helm Chart

### PR DESCRIPTION
**Pull request by joshleecreates**
Fixes https://github.com/open-telemetry/opentelemetry-helm-charts/issues/397

The first commit could be made a separate PR, just some small fixes I noticed while working on this.

### CODE REVIEW

File: charts/opentelemetry-demo/templates/_pod.tpl [hunk 1]
```diff
@@ -106,7 +112,7 @@ Exclude email service and treat differently because the addr. for the email serv
 
 {{- if eq .name "product-catalog-service" }}
 - name: FEATURE_FLAG_GRPC_SERVICE_ADDR
-  value: {{ (printf "%s-featureflag-service:50031" $prefix ) }}
+  value: {{ (printf "%s-featureflag-service:50053" $prefix ) }}
```
Comment (TylerHelmuth) [inline charts/opentelemetry-demo/templates/_pod.tpl:115]: Is this related to the shipping service? If not, let's put it in its own PR.

**Review by TylerHelmuth — COMMENTED** [review]
Thanks for taking this. Have you been able to test locally and confirm the shipping services is working as expected?

**Comment by joshleecreates** [comment]
> Thanks for taking this. Have you been able to test locally and confirm the shipping services is working as expected?

This works locally when I rebuild the images with the latest from the app repo. I think the published images are out of date with recent changes to the shipping and quote services.

**Comment by TylerHelmuth** [comment]
> This works locally when I rebuild the images with the latest from the app repo. I think the published images are out of date with recent changes to the shipping and quote services.

In that case we should probably hold off on this PR until the demo's official images are up to date. We don't want to release a chart that relies on demo code that hasn't been released yet. Let me know if I've misunderstood anything.

**Comment by joshleecreates** [comment]
> > This works locally when I rebuild the images with the latest from the app repo. I think the published images are out of date with recent changes to the shipping and quote services.
> 
> In that case we should probably hold off on this PR until the demo's official images are up to date. We don't want to release a chart that relies on demo code that hasn't been released yet. Let me know if I've misunderstood anything.

That makes sense. I'll figure out what's going on tomorrow.

File: charts/opentelemetry-demo/templates/_pod.tpl (hunk 1 above)
Comment (joshleecreates) [reply charts/opentelemetry-demo/templates/_pod.tpl]: I thought this was unrelated and removed it, but we actually need this so that the checkout service will complete it's flow to shipping and quote services.
